docs/implementing-cards.md

```diff
@@ -1,47 +1,95 @@
+	- [Trait reactions](#Trait-reactions)
+	- [Multiple choice reactions](#Multiple-choice-reactions)
+	- [Paying additional costs for reactions](#Paying-additional-costs-for-reactions)
+	- [Limiting the number of uses](#Limiting-the-number-of-uses)
+ - [Game actions](#Game-actions)
+ 	- [List of game actions](#List-of-game-actions)
+ - [Events](#Events)
+ 	- [List of events](#List-of-events)
+ - [Language](#Language)
+
 ### Getting started
 
+We recommend using VScode for implementing a card as there are many snippets that will help you.
+Whenever we refer to **action**, we mean also _spell_ and _job_ actions. 
+Whenever we mention **triggered abilities**, we mean _reaction_, _spell_reaction_, _traitReaction_ (in the future there can be some kind of interrupts also)
+Whenever you need to perform some basic game action, such as discard a card, boot a card, join posse, call out a dude or other refer to the [Game actions](#Game-actions) section.
+Whenever you need to do something base on event, refer to the [Events](#Events) section for the event names. !!! M2 later will also add event parameters !!!
 To implement a card, follow these steps:
 
-##### 1. Create a file named after the card.
+#### 1. Create a file named after the card.
 
 Cards are organized under the `/server/game/cards` directory by grouping them by "\<cycle number>-\<pack code>". Each word in the file name should be capitalized.
 
 ```
-/server/game/cards/01-Core/NobleLineage.js
-/server/game/cards/02.2-TRtW/UnswornApprentice.js
+/server/game/cards/01-DTR/SunInYerEyes.js
+/server/game/cards/02.2-DD/ItsNotWhatYouKnow.js
 ```
 
 #### 2. Create a class for the card and export it.
 
-Character, location, event and attachment cards should be derived from the `DrawCard` class.
-
-Plot cards should be derived from the `PlotCard` class.
+Each card type has its own class and snippet.
+_Dude_ - snippet: `dudecard`, class: `DudeCard`
+_Deed_ - snippet: `deedcard`, class: `DeedCard`
+_Goods_ - snippet: `goodscard`, class: `GoodsCard`
+_Spell_ - snippet: `spellcard`, class: `SpellCard`
+_Action_ - snippet: `actioncard`, class: `ActionCard`
+_Outfit_ - snippet: `outfitcard`, class: `OutfitCard`
 
-Agenda cards should be derived from the `AgendaCard` class.
+In case you are using snippet, just type the snippet prefix (e.g. `dudecard`) and VSCode should give you suggestion. If not, press Ctrl+space.
 
-The card class should have its `code` property set to the unique card identifier for that card. You can find these combining the 2-digit pack number (01) and 3-digit card number (036), or on [thronesdb.com](https://thronesdb.com/) by looking at the URL for the specific card.
+The card class should have its `code` property set to the unique card identifier for that card. You can find these in `/server/game/cards/cards_summary.csv`, or on [dtdb.co](https://dtdb.co/) by looking at the URL for the specific card. Or by combining the 2-digit pack number (01) and 3-digit card number (035).
```

Comment:
cards_summary.csv should be updated to store all packs and cards now there are only packs till There Comes a  Reckoning

Reply:
Yes it should :)

Reply:
will be done separately as latest packs are not yet processed (they are not in database)